content/en/cloud_cost_management/account_filtering/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Account Filtering
+description: Control what accounts you want to ingest into Cloud Cost Management.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+- link: "/cloud_cost_management/custom"
+  tag: "Documentation"
+  text: "Learn about Custom Costs"
+- link: "/cloud_cost_management/datadog_costs"
+  tag: "Documentation"
+  text: "Learn about Datadog Costs"
+- link: "/cloud_cost_management/saas_costs"
+  tag: "Documentation"
+  text: "Learn about SaaS Cost Integrations"
+- link: "/cloud_cost_management/tag_pipelines"
+  tag: "Documentation"
+  text: "Learn about Tag Pipelines"
+---
+
+## Overview
+
+[Cloud Cost Management][1] enables you to ingest cloud cost data for your AWS accounts, including management accounts for your [AWS Organizations][7].
+
+Use Account Filtering to get granular control of what AWS member accounts you want to pull into [Cloud Cost Management][1]. You will not incur any additional costs for accounts that you filter out.
+
+Currently, Account Filtering is only available for [AWS][8] member accounts, but soon will be available for [Azure][10] subscriptions and [Google Cloud][11] projects.
+
+## Filter accounts
```

Comment:
There's 2 cases here, you can also configure account filtering when creating a new account which is a different process